packages/zoe/src/applyFeePurse.js

```diff
@@ -0,0 +1,78 @@
+// @ts-check
+
+import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
+
+/**
+ * Partially apply an already existing feePurse to Zoe methods.
+ *
+ * @param {ERef<ZoeServiceFeePurseRequired>} zoe
+ * @param {ERef<FeePurse>} feePurse
+ * @returns {ZoeService}
+ */
+const applyFeePurse = (zoe, feePurse) => {
+  return Far('zoeService', {
+    makeFeePurse: (...args) => E(zoe).makeFeePurse(...args),
+
+    // A feePurse is required
+    install: (bundle, _overriddenFeePurse) => E(zoe).install(bundle, feePurse),
```

Comment:
What I had in mind was more like the following pattern

```suggestion
const applyFeePurse = (zoe, defaultFeePurse) => {
  return Far('zoeService', {
    makeFeePurse: (...args) => E(zoe).makeFeePurse(...args),

    // A feePurse is required
    install: (bundle, feePurse = defaultFeePurse) => E(zoe).install(bundle, feePurse),
```

where the bound purse is the default, and the caller can override the default on any individual call if they provide an explicit argument.

Reply:
Ah yep, sorry, I was getting too tired while doing this. Your suggestion is the only way this would work.

Reply:
Ok, that should be fixed now with tests as well!